client/src/styles/global.scss

```diff
@@ -1,2 +1,24 @@
-@import "~@trussworks/react-uswds/lib/uswds.css";
 @import "~@trussworks/react-uswds/lib/index.css";
+@import "~@trussworks/react-uswds/lib/uswds.css";
+
+.j40FloatLogoLeft {
```

Comment:
can we make this semantic? If we decide later to switch sides we'd need to update the class name